backend/api/Models/Parcel/PropertyModel.cs

```diff
@@ -19,7 +19,7 @@ public class PropertyModel : BaseModel
 
         public int AgencyId { get; set; }
 
-        public string SubAgency { get; set; }
+        public virtual string SubAgency { get; set; }
```

Comment:
Why virtual?  Are you overriding it somewhere?

Reply:
yeah, I could copy this entire model as-is into the Project namespace, but I thought I'd leave it and just create copies of the parcelModel and buildingModel and update those as needed.